components/onboarding_tasks/onboarding_tasks_manager.tsx

```diff
@@ -6,20 +6,23 @@ import {FormattedMessage} from 'react-intl';
 import {useDispatch, useSelector} from 'react-redux';
 import {matchPath, useLocation} from 'react-router-dom';
 
-import {trackEvent as trackEventAction} from 'actions/telemetry_actions';
+import {trackEvent, trackEvent as trackEventAction} from 'actions/telemetry_actions';
```

Comment:
Is this defined twice?

Reply:
That's a good catch. In the original code, there was a function argument in this file named `trackEvent` separate from the function call being imported here. For consistency, we might want to stick with `trackEventAction` for the call this PR adds, too.